recipes/spleeter-gpu/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "spleeter-gpu" %}
+{% set version = "1.4.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 49e776f26c561357eb9608a459030ab46cfe11edf21fb17470630a0d3777d453
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [osx or py<36 or py>37]
```

Comment:
```suggestion
  noarch: python
```

Reply:
As mentionner on previous suggestion `tensorflow-gpu` dependency does not have a distribution version for OSX platforms.

Reply:
If the package is noarch only Linux needs to pass and failures on other platforms are fine.

The Python version constraint can come from editing the `host` and `run` requirements.
```yaml
  - python >=3.6,<3.8
```

Reply:
Alright then, I will switch to noarch.

Reply:
But just to understand better the underlying process : since it is noarch, it means that an OSX user would be able to install it ? Even if it will not be able to run ?

Reply:
Only if they can also install the dependencies. I'd expect an error like `tensorflow-gpu is not available for platform osx-64`.

Reply:
Got it. I was expecting CI to pass for merge no matter what. But if it's ok to fail when noarch then I am ok with it :)

Reply:
Yeah, I've thought about it before but I'm not sure there is a good way to make the CI pass in staged-recipes for cases like this as the failures on other platforms sometimes expose real issues.

For future, when you ping the review teams its generally useful to say something like `it's noarch and fails on xxxxx because yyyyyy isn't available` to make the reviewers job easier.

Reply:
Good to know, I will next time :)